WordPress/Classes/ViewRelated/Me/App Settings/DebugMenuViewController.swift

```diff
@@ -114,7 +114,7 @@ private func showSuccessNotice() {
 
 private final class DebugMenuViewModel: ObservableObject {
     var blog: Blog? {
-        RootViewCoordinator.sharedPresenter.mySitesCoordinator.currentBlog
+        RootViewCoordinator.sharedPresenter.currentlyVisibleBlog()
     }
```

Comment:
This `blog` is not used and can be deleted.

Reply:
Done. It used to be needed for Quick Start, but now it's gone.